sql/src/main/java/io/crate/operation/collect/stats/StatsTablesService.java

```diff
@@ -191,12 +190,27 @@ private void setOperationsLogSink(int size, TimeValue expiration) {
         oldSink.close();
     }
 
+    private void setStatsEnabled(boolean enableStats) {
+        if (isEnabled == enableStats) {
```

Comment:
afaik this can never happen as the consumers are only triggered if something changes